test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua

```diff
@@ -0,0 +1,185 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "DISALLOWED"> to mobile app
+--
+-- Description:
+-- SDL doesn’t transfer not allowed "SetGlobalProperties"_request with valid ""autoCompleteList" param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>.
+-- 3. SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app
+-----------------------------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+require('user_modules/AppTypes')
+
+--[[ Local Functions ]]
+local registerAppInterfaceParams =
```

Comment:
@Aleksandriys, **local registerAppInterfaceParams** should be specified under --[[ Local variables ]]